client/src/Component/homepage/homepage.js

```diff
@@ -22,7 +22,17 @@ useEffect(()=>{
 
   return (
     <div className={dark ? "landing-page-orange dark-mode":"landing-page-orange"}>
-       
+    <div className="landing-page-orange">
```

Comment:
Where is this div over?

Reply:
@vedant-jain03 are you sure in your local your code working properly?

Reply:
Yes It Is working completely fine
I just want to mention that
Code is not clear at all thier is unnecesary code thier which can be converted into clean code

The code is really very messy
Means I have done my part successfully
But if you allow me I can change the code into clear and crystal format and more understandable code

Reply:
Like in code thier are so many divs even only for border thier is div
The navbar section is taking extra div which can be converted into one component

Reply:
Also in css 
For aligning items, left right is used
Which can be converted into one flex box

Reply:
@vedant-jain03 Ya, You can create a new issue about that.

Reply:
OK I will create a new issue on converting home page js and css into clean code